src/modules/limitOrders/services/tradeFlow/index.ts

```diff
@@ -50,33 +51,47 @@ export async function tradeFlow(
       signer: params.provider.getSigner(),
       validTo,
     })
-
-    logTradeFlow('LIMIT ORDER FLOW', 'STEP 4: presign order (optional)')
-    const presignTx = await (params.allowsOffchainSigning
-      ? Promise.resolve(null)
-      : presignOrderStep(orderId, params.settlementContract))
-
-    logTradeFlow('LIMIT ORDER FLOW', 'STEP 5: add pending order step')
+    logTradeFlow('LIMIT ORDER FLOW', 'STEP 4: add pending order step')
     addPendingOrderStep(
       {
         id: orderId,
         chainId: params.chainId,
         order: {
           ...order,
-          presignGnosisSafeTxHash: params.isGnosisSafeWallet && presignTx ? presignTx.hash : undefined,
+          isHidden: !params.allowsOffchainSigning,
```

Comment:
ubernit: it wouldn't have harmed a comment. I guess here, you only how as visible posted orders that are signed. presign flow don't include the signature in them. 

Maybe also nice to hint that in step 6 they are unhidden as long as `presignTx` succeeds